Datastores & Distributed File Systems/common/logger.py

```diff
@@ -0,0 +1,22 @@
+
+from datetime import datetime
+
+
+def log(func):
+    def wrapper(*args):
+
+        t1 = datetime.now()
+        result = func(*args)
+        t2 = datetime.now()
+
+        ts1 = datetime.timestamp(t1)
+        ts2 = datetime.timestamp(t2)
+
+        with open('output.txt', 'a') as log_file:
+            # FUNC_NAME|before time|after time|Time Lapse
+            log_file.write(
+                f'{func.__name__}|{args}|{t1}|{t2}|{float(ts2-ts1)}\n')
```

Comment:
This might slow down our ability to send a LOT of data fast by enough to adjust the results.

Thoughts?

Reply:
Maybe keep all the results in memory and then write them all to a file at the end?